pkg/deploy/spindeploy/spindeploy.go

```diff
@@ -44,22 +43,14 @@ func (d *Deployer) GetName() string {
 	return "spindeploy"
 }
 
-func (d *Deployer) isSpinnakerUpToDate(ctx context.Context, svc interfaces.SpinnakerService) (bool, error) {
```

Comment:
Why we are deleting this?

Reply:
This function was never called (dead code), seems that its contents was copied to `Deploy` function.